src/en/config-gce.md

```diff
@@ -1,255 +1,201 @@
-file, or output to stdout. In order to see the boilerplate environments.yaml on
-stdout you need to append the `--show` option. This is helpful if you have an
-existing environments.yaml and just need to add a section. E.g. 
-`juju generate-config --show`. You can then copy and paste the needed section.
+ - A Google account is required. See http://google.com .
 
-The generic configuration section generated for GCE will look
-something like this:
+ - A GCE account is required. See https://cloud.google.com/compute/ .
 
-```yaml
-    gce:
-      type: gce
-    
-      # Google Auth Info
-      # The GCE provider uses OAuth to authenticate. This requires that
-      # you set it up and get the relevant credentials. For more information
-      # see https://cloud.google.com/compute/docs/api/how-tos/authorization.
-      # The key information can be downloaded as a JSON file, or copied, from:
-      #   https://console.developers.google.com/project/<project>/apiui/credential
-      # Either set the path to the downloaded JSON file here:
-      auth-file:
-    
-      # ...or set the individual fields for the credentials. Either way, all
-      # three of these are required and have specific meaning to GCE.
-      # private-key:
-      # client-email:
-      # client-id:
-    
-      # Google instance info
-      # To provision instances and perform related operations, the provider
-      # will need to know which GCE project to use and into which region to
-      # provision. While the region has a default, the project ID is
-      # required. For information on the project ID, see
-      # https://cloud.google.com/compute/docs/projects and regarding regions
-      # see https://cloud.google.com/compute/docs/zones.
-      project-id:
-      # region: us-central1
-    
-      # The GCE provider uses pre-built images when provisioning instances.
-      # You can customize the location in which to find them with the
-      # image-endpoint setting. The default value is the a location within
-      # GCE, so it will give you the best speed when bootstrapping or adding
-      # machines. For more information on the image cache see
-      # https://cloud-images.ubuntu.com/.
-      # image-endpoint: https://www.googleapis.com
-```
+ - The Juju devel PPA (may change) is needed.
 
-!!! Warning: See [GH #747](https://github.com/juju/docs/issues/747) regarding
-configuration issues. This documentation should be fixed soon.
+ - The Juju client (the host running the below commands) will need the ability
+   to contact the GCE infrastructure on TCP ports 22 and 17070.
 
-To determine the values to add to the configuration, you will
-first have to set up a compatible project in GCE, by following these steps:
+Proceed to install the software.
 
+```bash
+sudo apt-add-repository -y ppa:juju/devel
+sudo apt-get update
+sudo apt-get install -y juju-core
+```
 
-## 1. Create a Google Compute Engine project
```

Comment:
why have you eschewed numbers/steps. people like numbers/steps, and sometimes it makes things easier for us too!